best-practice/definitions.ts

```diff
@@ -9,6 +9,56 @@ const github: readonly IBestPractice[] = [
     howToCheck: "Manual. View the repository on https://github.com",
     howToExempt: "N/A",
   },
+  {
+    name: "Branch Protection",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
+    description:
+      "Enable branch protection for the default branch, ensuring changes are reviewed before being deployed.",
+    howToCheck: "Manual. View the repository on https://github.com",
+    howToExempt: "N/A",
+  },
+  {
+    name: "Access",
+    owner: "[@guardian](https://github.com/orgs/guardian/teams/all)",
```

Comment:
What does owner @guardian mean?

Reply:
> What does owner @guardian mean?

I've used it to mean "everyone", rather than a specific team.